cpp/net/connection_pool.cc

```diff
@@ -188,7 +188,7 @@ ConnectionPool::ConnectionPool(libevent::Base* base)
 
 // static
 evhtp_res ConnectionPool::Connection::ConnectionErrorHook(
-    evhtp_connection_t* conn, evhtp_error_flags errtype, void* arg) {
+    evhtp_connection_t* conn, evhtp_error_flags, void* arg) {
```

Comment:
Actually, it turns out to always be some unhelpful value like "unknown" or whatever, or at least it was when I was writing the SSL stuff.

And if you want it, you can always put it back in while you're debugging and remove it once you're done.